ark/connection.py

```diff
@@ -7,56 +7,22 @@
 
 
 class Connection(object):
-    _nethash = None
-    _version = None
 
-    def __init__(self, client, host, port=None, nethash=None, version=None):
-        self.session = requests.Session()
+    def __init__(self, hostname, api_version_number):
+        if not isinstance(api_version_number, str) or api_version_number not in ['1', '2']:
+            raise Exception('Only versions "1" and "2" are supported')
 
-        self.host = host
-        self.port = port
-        self.nethash = nethash
-        self.version = version
-        self.client = client
+        self.session = requests.Session()
+        self.hostname = hostname
 
         self.session.headers.update({
             'port': '1',
-            'API-Version': self.client.api_version.replace('v', '')
+            'API-Version': api_version_number,
+            'nethash': '578e820911f24e039733b45e4882b73e301f813a0d2c31330dafda84534ffa23',
```

Comment:
based on our conversation ... should this be removed?

Reply:
Good spot. I've used this for testing out an endpoint that I had issues with. Removed now.